dts/bindings/sensor/ti,tmp1075.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2024 Arrow Electronics.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  TMP1075 Digital Temperature Sensor.  See more info at
+  https://www.ti.com/product/TMP1075
+
+compatible: "ti,tmp1075"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  conversion-rate:
+    description: Conversion rate in ms
+    type: int
+    default: 220
+    enum:
+      - 275 # 27.5ms
+      - 55
+      - 110
+      - 220
+  alert-gpios:
+    type: phandle-array
+    description: |
+      Identifies the ALERT signal, which is active-low open drain when
+      produced by the sensor.
+  one-shot-conversion:
+    description: One-shot conversion mode.
+    type: boolean
+  consecutive-fault-measurements:
+    description: Consecutive fault measurements to trigger the alert.
+    type: int
+    default: 4
+    enum:
+      - 1
+      - 2
+      - 3
+      - 4
+  alert-pin-active-high:
+    description: Polarity of the alert pin
+    type: boolean
+  interrupt-mode:
+    description: Selects the function of the ALERT pin
```

Comment:
mode and boolean seem kinda odd, this sounds more like an enum; if there are 2 classes then explain them in description.

Reply:
its either default or interrupt mode - it should be clear when you have sensor datasheet handy, otherwise all these parameters are like some kind of magic switches